app/main.py

```diff
@@ -32,13 +32,13 @@
 sentry_sdk.init(
     dsn=sentry_config['dsn'],
     release=sentry_config['release'],
-    environment=sentry_config['environment']
+    environment=sentry_config['environment'],
+    integrations=[
+        GraphqlIntegration(),
+    ],
 )
-# Ignore graphql.execution.utils to prevent duplicate Sentry events. Exceptions are handled by GraphQLSentryMiddleware.
-sentry_sdk.integrations.logging.ignore_logger("graphql.execution.utils")
```

Comment:
This was added a long time ago when we used a different GraphQL server library. I tried removing it and don't see duplicate events.